src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -95,6 +96,13 @@ private CallbackResponse handleDefendAllClaim(CallbackParams callbackParams) {
                 .build();
         }
         if ("DEFENDANT_RESPONSE_SPEC".equals(callbackParams.getRequest().getEventId())) {
+
+            if (caseData.getRespondToClaim().getHowMuchWasPaid().compareTo(caseData.getTotalClaimAmount())<0){
```

Comment:
A null check should be done on caseData.getRespondToClaim().getHowMuchWasPaid()

Also could we have a space on either side of the conditional operator please, just makes reading easier

Reply:
Done